src/pbcontroller.cpp

```diff
@@ -209,7 +209,7 @@ int PbController::initialize(int argc, char* argv[])
 		case ':':
 		case '?':
 			print_usage(argv[0]);
-			return EXIT_FAILURE;
+			exit(EXIT_FAILURE);
```

Comment:
I would expect an exit with `EXIT_SUCCESS` return code (feels more natural and makes it consistent with the return code for `-h`)

Reply:
I disagree: if a user supplies an unsupported option, that's an error, and the exit code should reflect that. I just checked various tools, and `cat -c`, `vim -j`, `bash -d` all return a non-zero exit code.

Is that persuasive enough? :) I've looked at [GNU Coding Standards](https://www.gnu.org/prep/standards/html_node/Command_002dLine-Interfaces.html) for CLI, but they don't mention exit codes. I wonder where I'd find a (somewhat) definitive answer to how programs should handle unknown options.

Reply:
I guess I misunderstood this piece of code.
I thought that `case '?':` would only get hit if the user called podboat like `podboat -?`.

After reading the relevant part of the `getopt_long()` documentation, I fully agree with you :)